src/pages/vaults/VaultInfoSections.tsx

```diff
@@ -105,21 +105,38 @@ const $DetailCard = styled.div`
 `;
 export const VaultDescription = ({ className }: { className?: string }) => {
   const stringGetter = useStringGetter();
-  const { vaultOperatorLearnMore } = useURLConfigs();
+  const { vaultOperatorLearnMore, vaultLearnMore } = useURLConfigs();
   const operatorName = useEnvConfig('megavaultOperatorName');
   return (
-    <div className={className} tw="text-color-text-0 font-small-medium">
-      {stringGetter({
-        key: STRING_KEYS.VAULT_DESCRIPTION,
-        params: {
-          OPERATOR_NAME: operatorName,
-          OPERATOR_LEARN_MORE: (
-            <Link isInline withIcon href={vaultOperatorLearnMore}>
-              {stringGetter({ key: STRING_KEYS.LEARN_MORE })}
-            </Link>
-          ),
-        },
-      })}
+    <div className={className} tw="flex flex-col gap-0.5 text-color-text-0 font-small-medium">
+      <p>
+        {stringGetter({
+          key: STRING_KEYS.VAULT_DESCRIPTION,
+        })}{' '}
```

Comment:
nit: can we get rid of the empty space

Reply:
I either need a space or to add margin. Space feels better so it is consistent with surrounding text.